website/docs/mutation.md

```diff
@@ -64,7 +64,7 @@ match:
   excludedNamespaces: []
 ```
 
-Note that the `applyTo` field is required for `Assign` and `ModifySet` mutators, and does not exist for `AssignMetadata` mutators.
+Note that the `applyTo` field is required for `Assign`, `AssignImage`, and `ModifySet` mutators, but does not exist for `AssignMetadata` mutators.
```

Comment:
Can we change this text to "all mutators but `AssignMetadata`", to lessen the likelihood of it going stale in the future?

Reply:
good idea. done.